packages/ui/src/forms/RenderFields/index.tsx

```diff
@@ -116,6 +118,7 @@ export const RenderFields: React.FC<RenderFieldsProps> = (props) => {
               }
               readOnly={isReadOnly}
               schemaPath={schemaPath}
+              siblingField={siblingField}
```

Comment:
Instead of doing this, just import the timezone options directly from payload. No prop is needed, you could do this from right inside the DateField component.

Reply:
Otherwise, if you need to pull the options from the field itself, like to support overrides in the future, sending the options directly as a prop is probably still a better pattern here as opposed to the entire sibling field.